conftest.py

```diff
@@ -122,3 +123,36 @@ def pytest_configure(config):
     #
     # Somehow the default database is always in memory, though.
     settings.DATABASES['transient']['TEST_NAME'] = ':memory:'
+
+
+@pytest.yield_fixture()
+def installed_packages_catalog():
+    with mock.patch('ideascube.serveradmin.catalog.Catalog', spec=True) as MockClass:
```

Comment:
> One important change is that it start/stop the mock automatically.

Right, of course, it does that for the lifecycle of the test function.

See, I'm glad I made this seemingly useless comment, because it allwed you to remind me of something I had forgotten. :laughing: